apache/incubator-pekko-sbt-paradox
# Add `assets.hostname` property in header and nav partials

Related to https://github.com/apache/incubator-pekko-site/issues/95 suppress https://github.com/apache/incubator-pekko-sbt-paradox/pull/126


There are link in the paradox theme where we don't prepend the `assets.hostname` property. This cause some 404 on the assets of https://pekko.apache.org. For example:


- https://pekko.apache.org/docs/pekko/current/
- https://pekko.apache.org/docs/pekko-http/current/

## Test locally
To test this locally:
1. Build and run `sbt publishLocal`
2. Specify the snapshot version in a downstream projects (e.g.: `incubator-pekko-http`, `incubator-pekko-grpc`)
3. Check the `target/paradox/site/main/` folder
